internal/cli/atlas/backup/restores/start.go

```diff
@@ -165,6 +165,8 @@ The following example creates a download restore:
          --clusterName myDemo \
          --snapshotId 5e7e00128f8ce03996a47179`, cli.ExampleAtlasEntryPoint()),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
+			opts.method = args[0]
```

Comment:
main change, this line was misplaced affecting all required validations to not run